### PR TITLE
Feature/add season controls on admin page

### LINKED ISSUE #55: Add 'Start new season' functionality to admin
Admins should be able to start a new season from the admin page.

The admin should be able to
  - [x] Create a new season
  - [x] Edit an existing season
  - [x] Delete an existing season

### PR DESCRIPTION
**Pull request by Kechr**
Closes #55 

image

### CODE REVIEW

File: src/db/queries/seasonQueries.ts [hunk 1]
```diff
@@ -15,3 +15,16 @@ export const getSeason = async (id: number) => {
     where: eq(seasonsTbl.id, id),
   });
 };
+
+export const getSeasons = async () => {
+  return await readDb.select().from(seasonsTbl);
```
Comment (Faustvii) [inline src/db/queries/seasonQueries.ts:20]: Normally we use the syntax `readDb.query.userTbl.findMany()`

File: src/pages/admin/Admin.tsx [hunk 2]
```diff
@@ -124,7 +134,88 @@ export const Admin = new Elysia({
         match_id: t.Number(),
       }),
     },
-  );
+  )
+  .post(
```
Comment (Faustvii) [inline src/pages/admin/Admin.tsx:138]: This could be a put no?

File: src/db/queries/seasonQueries.ts (hunk 1 above)
Comment (Kechr) [reply src/db/queries/seasonQueries.ts:20]: Hmm - is this to make it easier if we want to include relations? 
I looked at the documentation for selecting all columns in a table, and they provided this method.
https://orm.drizzle.team/docs/select#select-with-all-columns

File: src/pages/admin/Admin.tsx (hunk 2 above)
Comment (Kechr) [reply src/pages/admin/Admin.tsx:138]: We are using post method in the form, just like when editing a match, and it won't work with put I'm afraid.

**Review by Faustvii — APPROVED** [review]

**Pull request merged**